resources/js/components/Tool.vue

```diff
@@ -46,19 +75,40 @@ import formatters from '../mixins/formatters'
 
 export default {
     mixins: [formatters],
-
-    data: () => {
-        return {
-            loading: false,
-            jobs: [],
-        }
-    },
+    
+    data: () => ({     
+        jobs: [],
+        loading: false,
+        dispatchJob: null,
+        confirmDispatchJobModal: false,
+    }),
 
     mounted() {
         this.fetchJobs()
     },
 
-    methods: {
+    methods: {        
+
+        canDispatchCommand(command) {
+            return command.includes("\Jobs");
+        },
+
+        openConfirmationModal(job) {
+            this.dispatchJob = job;
+            this.confirmDispatchJobModal = true;
+        },
+        
+        confirmDispatchJob() {
+            const job = this.dispatchJob;
+            Nova.request().post('/nova-vendor/llaski/nova-scheduled-jobs/dispatch-job', { command: job.command })
+                .then((response) => {
+                    console.log({response});
```

Comment:
@tanjemark Let's show a toasted success and error message here. I think they show by default for some cases but not all.